src/calculateRentalCost.js

```diff
@@ -28,20 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  const dailyRent = 40;
-  const dailyCost = (dailyRent * days);
+  const dailyCost = (40 * days);
+  const weeklyDiscount = dailyCost - 20;
+  const monthlyDiscount = dailyCost - 50;
```

Comment:
that's not the discount 😄 
Take a deeper look at what is weekly/monthly discount